### PR TITLE
Add audio property to easter eggs (honk one, specifically)

### PR DESCRIPTION
**Pull request by FjorgeTheHuman**
Honk!

### CODE REVIEW

**Comment by Oj18** [comment]
No offence, but an added ~19KB per build isn't really worth it.

**Comment by FjorgeTheHuman** [comment]
I want to leave the toast in, and it's been updated so that it links to a file instead

File: src/ui/easterEggs.js [hunk 1]
```diff
@@ -47,9 +48,13 @@ export default {
         }
 
         goosemodScope.showToast(e.message);
+        
+        if (e.audio) {
+          new Audio(e.audio).play();
```
Comment (Oj18) [inline src/ui/easterEggs.js:53]: Just leaving this as `new Audio(...).play()` results in a memory leak.

**Review by Oj18 — CHANGES_REQUESTED** [review]

File: src/ui/easterEggs.js (hunk 1 above)
Comment (Oj18) [reply src/ui/easterEggs.js:53]: Best to store the `new Audio(...)` play it, then after it's played remove the element.

Comment (Oj18) [reply src/ui/easterEggs.js]: Resolved by latest PR commit.

**Review by Oj18 — APPROVED** [review]

**Review by FjorgeTheHuman — COMMENTED** [review]
maybe use something that isn't const for declaring the audio variable

**Comment by Oj18** [comment]
Alright, looks good now.

**Pull request merged**